apple/RNCWebView.m

```diff
@@ -46,6 +46,21 @@ -(id)inputAccessoryView
   return nil;
 }
 @end
+@interface RNCWKWebView_ : WKWebView
+@property (nonatomic, copy) NSArray<NSDictionary *> * _Nullable menuItems;
+@end
+@implementation RNCWKWebView_
+- (BOOL)canPerformAction:(SEL)action 
+              withSender:(id)sender{
+  
+  if (!self.menuItems || self.menuItems.count == 0) {              
+    return YES;
+  }
+  else{
```

Comment:
Since you are returning a value in the first condition, you don't need else here.

Once it's passed the first check, you can just return `NO`.

```
if (!self.menuItems || self.menuItems.count == 0) {              
    return YES;
}

return NO;
```